mpa/modules/datasets/seg_unlabeled_dataset.py

```diff
@@ -0,0 +1,34 @@
+from mmseg.datasets import DATASETS, build_dataset
+import numpy as np
+from mpa.utils.logger import get_logger
+
+from mmseg.datasets import CustomDataset
+
+logger = get_logger()
+
+
+@DATASETS.register_module()
+class UnlabeledSegDataset(CustomDataset):
```

Comment:
Do we need this dataset? I think this dataset class is not needed if we use ComposedDL. How do you think @harimkang

Reply:
I made this for implementing cutmix method for the next task. Making unlabeled_dataset is easiest way to generate 2 images for cutmix. So I left as this.

Reply:
I don't think we need this class only for cutmix. We can implement cutmix with 1 unlabeled loader with ComposedDL.

Reply:
I don't know I understand your point exactly, but if we impelement cutmix like you said, ComposedDL should be motified for seg cutmix, which is used across all the tasks, even other tasks does not use cutmix. I think it's better to deal it inside the dataset, not the loader.

Reply:
I don't understand either. All what this dataset do - create dataset like for the supervised one and iter it. What the reason of doing this? How it will help to implement cutmix?
We may build `unlabeled_dataset = build_dataset(dataset_cfg)` and use it as is. Then - mix unlabeled images in the batch during training for the cutmix.